resources/views/v4/admin/dashboard.blade.php

```diff
@@ -0,0 +1,15 @@
+@extends('v4.layouts.app')
```

Comment:
there is no need to put these in /v4 directory. just resources/views/admin/... is preferred. I'm in the process of cleaning up the unused stuff, and there is nothing from v2/v3 since those are on different servers